src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+class Car {
+	private final String name;
+	private int position;
+
+	public Car(String car) {
+		if (car.length() > 5) {
+			throw new IllegalArgumentException("이름은 5자리를 넘을 수 없다.");
+		}
+		this.name = car;
+	}
```

Comment:
```suggestion
	public Car(String name) {
		if (car.length() > 5) {
			throw new IllegalArgumentException("이름은 5자리를 넘을 수 없다.");
		}
		this.name = name;
	}
```

`name` 이 좀 더 어울리지 않을까요?